service/src/main/kotlin/fi/espoo/evaka/reservations/ReservationQueries.kt

```diff
@@ -540,17 +563,20 @@ SELECT pcd.child_id,
         -- reservation roll up
         (SELECT coalesce(jsonb_agg(json_build_object(
                'start', s.start_time,
-               'end', s.end_time)), '[]'::jsonb)
-        FROM (select ar.start_time, ar.end_time
-              FROM attendance_reservation ar
+               'end', s.end_time,
+               'staffCreated', s.staff_created)), '[]'::jsonb)
+        FROM (select ar.start_time, ar.end_time, eu.type = 'EMPLOYEE' as staff_created
+                FROM attendance_reservation ar
+                LEFT JOIN evaka_user eu ON ar.created_by = eu.id
```

Comment:
created_by has not null so LEFT is unnecessary

Reply:
Fixed!